### PR TITLE
Add https portal 600s timeouts to https portal component

### PR DESCRIPTION
**Pull request by pgerstoft**
The defaults for https-portal: https://github.com/SteveLTN/https-portal#configure-nginx-through-environment-variables

I am adding these to docker-compose.yml so that it will be easier to find in the future. It has been difficult to find the https-portal repo and to figure out how to configure its nginx.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -113,6 +113,10 @@ services:
     env_file: ./docker.env
     environment:
       STAGE: 'local' # <- Change 'local' to 'production' to use a LetsEncrypt signed SSL cert
+      KEEPALIVE_TIMEOUT: 65
+      PROXY_CONNECT_TIMEOUT: 60
+      PROXY_SEND_TIMEOUT: 60
+      PROXY_READ_TIMEOUT: 60
```
Comment (pgerstoft) [inline docker-compose.yml:119]:
```suggestion
      KEEPALIVE_TIMEOUT: 125
      PROXY_CONNECT_TIMEOUT: 120
      PROXY_SEND_TIMEOUT: 120
      PROXY_READ_TIMEOUT: 120
```

**Comment by kyle-retool** [comment]
Should make these timeouts really large? Maybe start with 10 minutes? That way we don't get bit if people up `DBCONNECTOR_QUERY_TIMEOUT_MS` to a larger value?

File: docker-compose.yml [hunk 2]
```diff
@@ -113,6 +113,10 @@ services:
     env_file: ./docker.env
     environment:
       STAGE: 'local' # <- Change 'local' to 'production' to use a LetsEncrypt signed SSL cert
+      KEEPALIVE_TIMEOUT: 125
+      PROXY_CONNECT_TIMEOUT: 120
+      PROXY_SEND_TIMEOUT: 120
+      PROXY_READ_TIMEOUT: 120
```
Comment (pgerstoft) [inline docker-compose.yml:119]:
```suggestion
      KEEPALIVE_TIMEOUT: 605
      PROXY_CONNECT_TIMEOUT: 600
      PROXY_SEND_TIMEOUT: 600
      PROXY_READ_TIMEOUT: 600
```

**Review by rhinon — APPROVED** [review]
LGTM

**Pull request merged**